scapy/contrib/gtp.py

```diff
@@ -258,19 +258,21 @@ class GTP_U_Header(GTPHeader):
     def guess_payload_class(self, payload):
         # Snooped from Wireshark
         # https://github.com/boundary/wireshark/blob/07eade8124fd1d5386161591b52e177ee6ea849f/epan/dissectors/packet-gtp.c#L8195  # noqa: E501
+        if self.E == 1:
+            if self.next_ex == 0x85:
+                return GTPPDUSessionContainer
+            else
```

Comment:
This triggers PEP08 errors. See https://travis-ci.com/secdev/scapy/jobs/269661800#L655

Reply:
I fixed the style.